masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+      "required": true
+    },
+    "status": { "$ref": "#/$defs/status", "required": true },
+    "returnTotals": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "required": true,
+            "enum": ["items", "shipping", "tax"]
+          },
+          "value": {
+            "type": "integer",
+            "required": true
+          }
+        }
+      }
+    },
+    "customerProfileData": {
+      "type": "object",
+      "required": true,
+      "properties": {
+        "userId": {
+          "type": "string",
+          "required": true
+        },
+        "name": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string",
+          "format": "email"
+        },
+        "phoneNumber": { "type": "string", "maxLength": 50 }
+      }
+    },
+    "pickupReturnData": {
+      "type": "object",
+      "required": true,
+      "properties": {
+        "addressId": {
+          "type": "string"
+        },
+        "address": {
+          "type": "string",
+          "required": true
+        },
+        "city": {
+          "type": "string",
+          "required": true
+        },
+        "state": {
+          "type": "string",
+          "required": true
+        },
+        "country": {
+          "type": "string",
+          "required": true
+        },
+        "zipCode": {
+          "type": "string",
+          "required": true
+        },
+        "addressType": {
+          "type": "string",
+          "required": true,
+          "enum": ["pickup-point", "customer-address"]
+        },
+        "returnLabel": {
+          "type": "string"
+        }
+      }
+    },
+    "refundPaymentData": {
+      "type": "object",
+      "required": true,
+      "properties": {
+        "refundPaymentMethod": {
+          "type": "string",
+          "required": true,
+          "enum": ["bank", "card", "giftCard"]
+        },
+        "iban": {
+          "type": "string"
+        },
+        "accountHolderName": {
+          "type": "string"
+        },
+        "paidWithCc": {
+          "type": "boolean"
+        }
+      }
+    },
+    "items": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "orderItemIndex": { "type": "integer", "required": true },
+          "quantity": { "type": "integer", "required": true },
+          "verifiedItems": { "type": "integer" },
+          "reasonCode": { "type": "string", "required": true },
+          "reason": { "type": "string", "required": true },
+          "condition": {
+            "type": "string",
+            "required": true,
+            "enum": [
+              "newWithBox",
+              "newWithoutBox",
+              "usedWithBox",
+              "usedWithoutBox"
+            ]
+          }
+        }
+      }
+    },
```

Comment:
For `items`, I removed all the product information (like image url, skuId, unit price, etc), since we can get it from the directly from the order. To match that, I added the `orderItemIndex`, to keep track of the product inside `order.items`. It might be useful for split orders, or when a product is given for free when buying two units for that, for example.

`verifiedItems` will be filled after the store has verified and approved the return for that item. I am still not convinced we need the field here, since we can get that information from `refundData.items` property (we just need to check if the verify process and the invoice process happens at the same time). We may delete it in the future.